novelwriter/dialogs/about.py

```diff
@@ -187,6 +187,7 @@ def _fillAboutPage(self):
                 ("Português", "Bruno Meneguello"),
                 ("简体中文", "Qianzhi Long"),
                 ("Español Latinoamericano", "Tommy Marplatt"),
+                ("Nederlands", "Martijn van der Kleijn (mvdkleijn)"),
```

Comment:
I prefer if you just list your name here, and not your username. For Jan Lüdke the part in parentheses is his online alias, which is fine to add if you have one.

Reply:
Done